### ISSUE TITLE
[Feature Request] more options for ordering the model list

### ISSUE THREAD

**Issue by hayden-fr** [hayden-fr/ComfyUI-Model-Manager#70]
What sorting methods are expected to be needed?

**Comment by hayden-fr** [comment]
Optimizing the search function may be more practical

**Comment by Ainaemaet** [comment]
Mostly just the ability to click the current selector (name, date created, largest, etc.) and have it switch from ascending to descending.   

If you want to swing it, something like 'most used' would be pretty awesome I think.

Maybe the ability to select the folder and sort only in a specific folder would be wonderful as well for those of us with lots of nested folders for organization.

**Comment by hayden-fr** [comment]
How about adding a favorite feature?

Place your favorite models at the front, or add a new type to list separately.

**Comment by Ainaemaet** [comment]
Yup - I was actually going to put that down there but got sidetracked and had a brain fart lol. 

I think that is a wonderful idea.
